### DISCUSSION TITLE
Unable to Import Software Update - Compliance by Collection

### DISCUSSION THREAD

**Discussion by mchippy** [SCCM-Zone/CM-SRS-Dashboards#33] [General]
I installed the newer version (5.1.2) in a new client's environment and everything went pretty smooth until the script went to import the report **Software Update - Compliance by Collection**.

SSRS is version 2019, so we meet the minimum criteria. 

It wasn't happy with the usage of "<" ">" and "&". After comparing the report for a previous version, I was able to update all the symbols to the &lt; &gt; and &amp; and that seems to have worked. Report was able to be imported.

**Comment by Ioan-Popovici** [comment]
Thanks, I'll have a look to see what's wrong there.

**Reply by Ioan-Popovici** [reply]
Yeah I need a test environment that works for these reports. Updating them blindly sucks. I'll fix the bugs ASAP.

**Comment by mchippy** [comment]
Well, knowing that - you do an amazing job doing it blindly! It was a simple replace after importing it and identifying the lines and characters it didn’t like. No biggie.

________________________________
